bandchain/chain.json

```diff
@@ -3,18 +3,55 @@
     "chain_name": "bandchain",
     "status": "live",
     "network_type": "mainnet",
-    "pretty_name": "Bandchain",
+    "pretty_name": "BandChain",
     "chain_id": "laozi-mainnet",
-    "bech32_prefix": "band",
     "daemon_name": "bandd",
+    "node_home": "$HOME/.band",
+    "bech32_prefix": "band",
     "genesis": {
-        "genesis_url": "https://raw.githubusercontent.com/ovrclk/net/master/mainnet/genesis.json"
+        "genesis_url": "https://raw.githubusercontent.com/bandprotocol/launch/master/laozi-mainnet/genesis.json"
     },
     "codebase": {
         "git_repo": "https://github.com/bandprotocol/chain",
-        "recommended_version": "v2.3.0",
+        "recommended_version": "v2.3.2",
         "compatible_versions": [
-            "v2.3.0"
+            "v2.3.2"
+        ]
+    },
+    "peers": {
+        "seeds": [
+            {
+                "id": "8d42bdcb6cced03e0b67fa3957e4e9c8fd89015a",
+                "address": "34.87.86.195:26656"
+            },
+            {
+                "id": "543e0cab9c3016a0e99775443a17bcf163038912",
+                "address": "34.150.156.78:26656"
+            }
+        ],
+        "persistent_peers": [
+            {
+                "id": "98823087b61d442a4ab86998709c77b2e517ee78",
+                "address": "35.240.152.216:26656"
+            },
+            {
+                "id": "3ea84babead3d6bc488810a0f2cf0744cf5c68fe",
+                "address": "34.86.22.251:26656"
+            }
+        ]
+    },
+    "apis": {
+        "rpc": [
+            {
+                "address": "http://rpc.laozi1.bandchain.org:80",
+                "provider": "bandprotocol"
+            }
+        ],
+        "rest": [
+            {
+                "address": "https://laozi1.bandchain.org/api",
```

Comment:
This endpoint seems to be unreachable

Reply:
Could you try again using this example? The endpoint seems to be working fine on my end. 
https://laozi1.bandchain.org/api/oracle/v1/active_validators

Reply:
It seems to be working fine on my end, could you try again using this example? https://laozi1.bandchain.org/api/oracle/v1/active_validators

Reply:
Oh interesting.  Okay thanks.  
Normally, the base endpoint shows something saying "not implemented".  But since this works, looks good!  Thanks!